source/scripts/Router.js

```diff
@@ -0,0 +1,32 @@
+// Router.js
+export class Router {
+    static routes = {};
+  
+    constructor(homeFunc) {
+      this['home'] = homeFunc;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group23/cse110-fa21-group23/pullRequest?prid=8422335)
```suggestion
      this["home"] = homeFunc;
```